configs/_config.checks.yml

```diff
@@ -0,0 +1,67 @@
+html_proofer:
+  file_ignore:
```

Comment:
Before merging, you'll need to remove all entries from the list to match the `master` branch. We merged several PRs today to increase test coverage and I want to be sure we continue checking those.

Reply:
sure